cmd/cli/version_test.go

```diff
@@ -131,47 +132,27 @@ func TestGetMeshVersion(t *testing.T) {
 	}
 }
 
-func TestOutputVersionInfo(t *testing.T) {
+func TestOutputPrettyVersionInfo(t *testing.T) {
 	tests := []struct {
-		name        string
-		versionInfo versionInfo
-		expected    string
-		clientOnly  bool
+		name                  string
+		remoteVersionInfoList []*remoteVersionInfo
+		expected              string
+		clientOnly            bool
```

Comment:
Since `outputPrettyVersionInfo` doesn't use `clientOnly`, it looks like we don't need to define it here.